app/code/Magento/Catalog/Model/LocatorService.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Model;
+
+class LocatorService
```

Comment:
let's remove this class as it looks like helper storing different methods just to avoid copy-paste

Reply:
LocatorService was removed